.github/workflows/_build.yaml

```diff
@@ -19,21 +19,27 @@ on:
         default: 'NOT SPECIFIED'
       ARTIFACT_NAME:
         type: string
-        description: 'Name of the artifact zip file'
-        required: false
-        default: 'artifact-pax-build'
+        description: 'Name of the artifact zip file, e.g. artifact-t5x-build'
+        required: true
       BADGE_FILENAME:
         type: string
-        description: 'Name of the endpoint JSON file for shields.io badge'
-        required: false
-        default: 'badge-pax-build'
+        description: 'Name of the endpoint JSON file for shields.io badge, e.g. badge-t5x-build'
+        required: true
+      CONTAINER_NAME:
+        type: string
+        description: "Container name, e.g. upstream-t5x"
+        required: true
+      DOCKER_FILE:
```

Comment:
Please use `DOCKERFILE` with no underscore in between.

Reply:
Done :)